src/layouts/app-layout/app-layout-component.jsx

```diff
@@ -2,13 +2,15 @@ import React, { PureComponent } from 'react';
 import Proptypes from 'prop-types';
 import FeaturedGlobe from 'pages/featured-globe';
 import DataGlobe from 'pages/data-globe';
-
+import MapIframe from 'pages/map-iframe';
 
 class App extends PureComponent {
   render() {
     const { route } = this.props;
     const { page } = route;
-    return page === 'featured-globe' ? <FeaturedGlobe /> : <DataGlobe />;
+    const embedded = page === 'map-iframe';
```

Comment:
Ok, this is working and I do not want to block this PR from being merged, but probably we will need to review the use cases for the embed map (as it seemed the other day that Chris is not using it any more, as he said to get rid of the `location` params under the share url).
Also the logic behind this page selection might not respond to the interactions currently available for the user on the platform (as we can also share the `iframe` on the `featuredGlobe` not only on the data one).
Maybe having a `embed` layout separated from the  from the `app-layout`?

Reply:
so, maybe it is the naming, but I thought there are two different use cases here:
- the `iframe` we have in the Share modal that we can embed anywhere and do everything we can do on the website,
- the `map-iframe` that has a separate path `/map` that is for Chris to do some storytelling with it (+ the pledges) and use `postRobot` for that. For that map-iframe it's true, it's using only the expert mode web scene. We will probably have to revisit it anyway after we know more what kind of storytelling they'd like to do when it comes to the featured mode.